services/install-bookstack.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/bookstack
+
+  # create yml(s)
+  cat << EOF > /srv/bookstack/bookstack.yml
+version: "2"
+services:
+  bookstack:
+    image: linuxserver/bookstack
+    container_name: bookstack
+    environment:
+      - PUID=\${PUID}
+      - PGID=\${PGID}
+      - DB_HOST=bookstack_db
+      - DB_USER=\${DB_USERNAME}
+      - DB_PASS=\${DB_PASSWORD}
+      - DB_DATABASE=\${DB_DATABASE}
+    volumes:
+      - /srv/bookstack:/root/.bookstack
+    ports:
+      - 6875:80
+    restart: unless-stopped
+    depends_on:
+      - bookstack_db
+  bookstack_db:
+    image: linuxserver/mariadb
+    container_name: bookstack_db
+    environment:
+      - PUID=\${PUID}
+      - PGID=\${PGID}
+      - MYSQL_ROOT_PASSWORD=\${DB_PASSWORD}
+      - TZ=Europe/London
+      - MYSQL_DATABASE=\${DB_DATABASE}
+      - MYSQL_USER=\${DB_USERNAME}
+      - MYSQL_PASSWORD=\${DB_PASSWORD}
+    volumes:
+      - /srv/bookstack:/root/.bookstack
+    restart: unless-stopped
+EOF
+
+  # create .env with default values
+  cat << EOF > /srv/bookstack/.env
+PUID=1000
+PGID=1000
+DB_USERNAME=db_username
+DB_PASSWORD=db_password 
+DB_DATABASE=bookstackapp
+EOF
+
+  # add autorun
+  cat << EOF > /srv/bookstack/autorun
+bookstack_autorun=true
+
+if [ "$bookstack_autorun" = true ]; then
+  treehouses services bookstack up
+fi
+
```

Comment:
If you could add an additional space here (for formatting issues) that would be great.

Reply:
Other than that, I got it up in about 2 minutes.